bin/tx-prover/README.md

```diff
@@ -120,6 +122,26 @@ Then access the Jaeger UI at `http://localhost:16686/`.
 
 If Docker is not an option, Jaeger can also be set up directly on your machine or hosted in the cloud. See the [Jaeger documentation](https://www.jaegertracing.io/docs/) for alternative installation methods.
 
+## Metrics
+
+The proxy includes a service that provides metrics for [Prometheus](https://prometheus.io/docs/introduction/overview/). This service is always enabled and uses the port defined in the configuration file. The metrics are available at the `/metrics` endpoint.
```

Comment:
Yes, because in the application we have the prometheus exporter. It is always serving metrics in `metrics_exporter_host:port/metrics` (e.g. `http://127.0.0.1:6192/metrics`), and do not care if anyone is requesting it or not.

Then a prometheus collector, the one mentioned in the comment, scraps periodically the endpoint and populates the DB with the metrics.

Finally, with Grafana we make queries to the prometheus DB and create graphics with that data.

Reply:
Being more specific on answering your question: yes, you can run the service without never scrapping the metrics.

Reply:
> Yes, because in the application we have the prometheus exporter. It is always serving metrics in `metrics_exporter_host:port/metrics` (e.g. `http://127.0.0.1:6192/metrics`), and do not care if anyone is requesting it or not.

To make sure I understood correctly: 
- We've added an endpoint (by default `http://127.0.0.1:6192/metrics`) to the service that returns the metrics we defined in Prometheus format. This endpoint is always there regardless of whether anything is using it or not.
- We can run Prometheus collector that will periodically ping this endpoint and collect data from it. This data then would be saved in a database.
- Then Grafana frontend would read the data from the database and visualize it according to the template we defined.

Is this correct?

Reply:
Yes, that's correct.